frontend/src/components/FilterTab.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./styles/FilterTab.scss";
+
+function FilterTab() {
+  return (
+    <div className="FilterTab">
+      <div className="center">
+        {/* GET CONCERNED TITLE TO INSERT IT THERE ( AMBASSADORS? CAUSES? ) */}
+        <p>Filtrer parmis les /* import name props */:</p>
```

Comment:
I think you didn't finish this line of code

Reply:
Done !